### ISSUE TITLE
Rerun initialization items if service worker installation and activation triggered

### ISSUE THREAD

**Issue by mglaman** [mglaman/wasm-drupal#72]
I think there is a bug where checking the existing session or automatic start fails if the service worker was just installed and activated.

If we hit the activation event we may need a refresh or manually run the event when DOM content is loaded

**Comment by mglaman** [comment]
We have a `await navigator.serviceWorker.ready` but it doesn't seem to work as I'm expecting: https://developer.mozilla.org/en-US/docs/Web/API/ServiceWorkerContainer/ready

The php-wasm demo checks if the controller is available https://developer.mozilla.org/en-US/docs/Web/API/ServiceWorkerContainer/controller

```
setTimeout(() => navigator.serviceWorker.controller || window.location.reload(), 350);
```

This quirk can be replicated with a force refresh.

**Comment by mglaman** [comment]
This seems like a good solution: https://stackoverflow.com/a/62596701/1949744

```
navigator.serviceWorker.getRegistration().then(function(reg) {
  // There's an active SW, but no controller for this tab.
  if (reg.active && !navigator.serviceWorker.controller) {
    // Perform a soft reload to load everything from the SW and get
    // a consistent set of resources.
    window.location.reload();
  }
});
```

**Comment by mglaman** [comment]
fixed by #105
